Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -367,7 +367,7 @@ static PermissionsPolicy::PolicyDirective parsePolicyDirective(StringView value,
 PermissionsPolicy::PolicyDirective PermissionsPolicy::processPermissionsPolicyAttribute(const HTMLIFrameElement& iframe)
 {
     auto allowAttributeValue = iframe.attributeWithoutSynchronization(allowAttr);
-    auto policyDirective = parsePolicyDirective(allowAttributeValue, iframe.document().securityOrigin().data(), declaredOrigin(iframe)->data(), iframe.document().quirks().shouldStarBePermissionsPolicyDefaultValue());
+    auto policyDirective = parsePolicyDirective(allowAttributeValue, iframe.document().securityOrigin().data(), declaredOrigin(iframe)->data(), false);
```

Comment:
Since this is the only caller of this algorithm as far as I can tell, you should really update the algorithm to remove the argument and its associated branching altogether.

Reply:
There are two functions that should be updated `parsePolicyDirective` and `parseAllowlist`. Both functions are only called in one place.

Reply:
I removed the true/false parameters and if check